connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/DataFrameWriter.scala

```diff
@@ -201,6 +201,22 @@ final class DataFrameWriter[T] private[sql] (ds: Dataset[T]) {
     this
   }
 
+  /**
+   * Clusters the output by the given columns on the file system. The rows with matching values in
```

Comment:
let's be a bit more general as data sources are not always based on file system. How about `... given columns on the storage.`?

Reply:
sure, updated here and below